src/main/java/roomescape/auth/controller/AuthController.java

```diff
@@ -0,0 +1,49 @@
+package roomescape.auth.controller;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.member.domain.Member;
+import roomescape.auth.service.AuthService;
+import roomescape.global.config.AuthMember;
+import roomescape.auth.dto.LoginCheckResponse;
+import roomescape.auth.dto.LoginRequest;
+import roomescape.auth.dto.TokenResponse;
+
+@RestController
+@RequestMapping("/login")
+public class AuthController {
+
+    private final AuthService authService;
+
+    public AuthController(final AuthService authService) {
+        this.authService = authService;
+    }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.OK)
+    public void login(
+            @RequestBody @Valid final LoginRequest loginRequest,
+            final HttpServletResponse response
+    ) {
+        final TokenResponse token = authService.createToken(loginRequest);
+        final Cookie cookie = new Cookie("token", token.accessToken());
+        cookie.setHttpOnly(true);
+        cookie.setPath("/");
+        response.addCookie(cookie);
+    }
+
+    @GetMapping("/check")
+    public ResponseEntity<LoginCheckResponse> loginCheck(@AuthMember final Member member) {
+        final LoginCheckResponse response = new LoginCheckResponse(member.getName());
+        return ResponseEntity.ok().body(response);
```

Comment:
ok 인자로 body 를 넘겨줄 수 있습니다 :)
```suggestion
        return ResponseEntity.ok(response);
```

Reply:
이 부분은 의도한 부분입니다!
두 방식 다 가능하지만 전자가 명시적으로 `body()`를 넘겨준다는 느낌이 있어서 전자로 표시했습니다!
또한 다른 컨트롤러에서도 전자로 작성해서 같은 방식을 일종의 컨벤션으로 사용했습니다 👍